recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
Thanks @ocefpaf ! We mean to skip Python 3 only on OS X. 

I initially looked at AppVeyor which seems to be building for py2 only. However now looking again I saw that CircleCI built for linux py2 and py3 and Travis built for osx py2. Is AppVeyor not expected to have a py3 build?